### PR TITLE
Report original AccessoryInformation if device is of type HomeKit Controller

### PR DESCRIPTION
**Pull request by TheRealLink**

### CODE REVIEW

**Comment by robertklep** [comment]
Thanks :D Can you provide a squashed commit?

**Comment by TheRealLink** [comment]
> Thanks :D Can you provide a squashed commit?

You are welcome :)

Do you have squash merging enabled? This should do the trick: https://docs.github.com/en/github/administering-a-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests

**Comment by robertklep** [comment]
The issue isn't with merging, the issue is that more than a single commit makes it difficult to review your changes. So if you can squash your commits, it should come out as a single commit which makes my life easier :D

**Comment by robertklep** [comment]
@TheRealLink can you take a look at the remark I posted about `CarbonMonoxideSensor`? If you can push a fix for that I'll test and merge your PR 👍🏻

**Comment by TheRealLink** [comment]
> @TheRealLink can you take a look at the remark I posted about `CarbonMonoxideSensor`? If you can push a fix for that I'll test and merge your PR 👍🏻

Sure thing, but unfortunately there is no remark present ;)

**Review by robertklep — CHANGES_REQUESTED** [review]

**Comment by robertklep** [comment]
@TheRealLink how about now? 😅

File: lib/devices/sensor.js [hunk 1]
```diff
@@ -127,45 +126,43 @@ module.exports = function(device, api, capabilities) {
...
-    homekitAccessory
-      .addService(Service.CarbonMonoxideSensor, device.name)
-      .getCharacteristic(Characteristic.CarbonMonoxideDetected)
-      .on('get', function(callback) {
-        callback(null, ~~capValue('alarm_co'));
-      });
-  }
+  if ('measure_co' in capabilities || 'alarm_co' in capabilities) {
+    var coService = homekitAccessory
+      .addService(Service.CarbonDioxideSensor, device.name);
```
[the remark this answers is not in the document]
Comment (TheRealLink) [reply lib/devices/sensor.js]: Done!

**Comment by robertklep** [comment]
I just published v3.2.0 to the test channel: https://homey.app/a/com.swttt.homekit/test/

**Pull request merged**